DSharpPlus/Entities/DiscordWebhookBuilder.cs

```diff
@@ -187,5 +218,23 @@ public DiscordWebhookBuilder AddMentions(IEnumerable<IMention> mentions)
             this._mentions.AddRange(mentions);
             return this;
         }
+
+
+        public async Task<DiscordMessage> SendWebhook(DiscordWebhook webhook)
```

Comment:
This should be renamed to `SendAsync()` to match consistency with the other builder.

Reply:
done